static/scss/detail/enterprise/learning-journey.scss

```diff
@@ -0,0 +1,119 @@
+// sass-lint:disable mixins-before-declarations
+
+.learning-journey-block {
+  padding-top: 50px;
+  background: linear-gradient(180deg, #f4f6f7 0%, #fff 100%);
+  position: relative;
+
+  @include media-breakpoint-down (sm) {
+    padding: 30px 5px 0px 5px;
+  }
+
+  h1 {
+    color: var(--BaseDark, #000);
+    text-align: center;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+
+    @include media-breakpoint-down (sm) {
+      text-align: initial;
+      font-size: 24px;
+      font-weight: 400;
+    }
+  }
+
+  .learning-journey-bg {
+    position: absolute;
+    bottom: 0px;
+    max-width: 100%;
+  }
+}
+
+.learning-journey-container {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+
+  @include media-breakpoint-down (sm) {
+    display: block;
+    margin-top: 25px;
+  }
+
+  .learning-journey-content {
+    display: flex;
+    flex-direction: column;
+    width: 36%;
+
+    @include media-breakpoint-down (sm) {
+      width: 100%;
+    }
+
+    ul {
+      padding: 0px;
+    }
+
+    li {
+      list-style-type: none;
+      border-radius: 5px;
+      background: #fff;
+      padding: 12px 18px;
+      box-shadow: 0px 2px 5px 0px rgba(0, 0, 0, 0.05);
+      color: var(--BaseDark, #000);
```

Comment:
We should create a variable for #000 in variables.scss like all other variables and use it here.